kind/kind.sh

```diff
@@ -48,42 +41,18 @@ function _wait_kind_up {
     kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
 }
 
-function _fetch_kind() {
-    mkdir -p "${KIND_DIR}"
-    KIND="${KIND_DIR}"/.kind
-    if [ -f "$KIND" ]; then
-        current_kind_version=$($KIND --version | awk '{print $3}')
-    fi
-    if [[ $current_kind_version != $KIND_VERSION ]]; then
-        echo "Downloading kind v$KIND_VERSION"
-        if [[ "$OSTYPE" == "darwin"* ]]; then
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-darwin-${ARCH} -o "$KIND"
-        else
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-linux-${ARCH} -o "$KIND"
-        fi
-        chmod +x "$KIND"
-    fi
-}
-
```

Comment:
imho, we shouldn't remove this rather use the approach in #31  to validate if `kind exists` and the version is as expected.

Reply:
@sthaha , the key idea for this PR is not statically or ENV-based define the kind version, we just need to transfer the `kind` installation effort to runner admin, just as `Github-hosted runners` do. For developers, he/she needs to do one-shot installation of `kind` and `kubectl` when he setup his/her test machine at the very beginning.

Reply:
I agree about the installation part, but the validation for the versions must be performed in the script to ensure everyone stays on the same version. We can also add  `--no-validate  or --ci` flag to suppress unnecessary validations in CI.